psyneulink/core/components/functions/function.py

```diff
@@ -627,10 +627,10 @@ def _get_current_function_param(self, param_name, context=None):
                                 f"the current value of a function parameter. 'variable' is not a function parameter. "
                                 f"If looking for {self.name}'s default variable, try {self.name}.defaults.variable.")
         try:
-            return self.owner._parameter_ports[param_name].parameters.value._get(context)
+            return self.owner._parameter_ports[param_name].parameters.value.get(context)
         except (AttributeError, TypeError):
             try:
-                return getattr(self.parameters, param_name)._get(context)
+                return getattr(self.parameters, param_name).get(context)
```

Comment:
should this get @kmantel 's review?

Reply:
yes - will revert for now (ideally should be a separate issue)